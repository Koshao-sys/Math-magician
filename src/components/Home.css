.home-container {
  width: 70%;
  margin: auto;
}
