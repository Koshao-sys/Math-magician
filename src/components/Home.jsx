import React from 'react';
import './Home.css';

const Home = () => {
  const paragragh = 'The Math Magician site is a fascinating platform that allows users to perform'
    + 'complex mathematical calculations with ease.The site features a user-friendly'
    + 'calculator that can handle a wide range of mathematical operations, from basic'
    + 'addition and subtraction to advanced trigonometry and calculus. The calculator'
    + 'is highly responsive, ensuring that users can enter their equations quickly and'
    + 'easily, and get instant results. But the Math Magician site is more than just a'
    + 'calculator. It also features a unique API that delivers inspiring and thought-provoking'
    + 'quotes from some of the worlds most renowned mathematicians. The quotes are displayed'
    + 'on the sites screen, providing users with a source of inspiration and motivation as'
    + 'they work through their calculations. Whether you are a student, a teacher, or a professional'
    + 'mathematician, these quotes can help you stay focused, engaged, and inspired as you tackle'
    + 'even the toughest mathematical problems.One of the most impressive features of the Math'
    + 'Magician site is its versatility. Whether you are working on a homework assignment,'
    + 'preparing for an exam, or simply exploring the fascinating world of mathematics,'
    + 'this platform has something to offer.The calculator is highly adaptable, making it'
    + 'suitable for everyone from beginners to experts, and the APIs vast collection of'
    + 'quotes ensures that there is always something new and interesting to discover.'
    + 'Overall, the Math Magician site is a valuable resource for anyone interested in'
    + 'mathematics. Its calculator is highly responsive and user-friendly, while its API'
    + 'provides a wealth of inspiration and motivation for anyone looking to explore the'
    + 'fascinating world of mathematics. Whether you are looking to solve complex equations,'
    + 'brush up on your calculus, or simply immerse yourself in the beauty of mathematics,'
    + 'the Math Magician site is an excellent place to start.';

  return (
    <div className="home-container">
      <h1>Welcome to our page!</h1>
      <p>{paragragh}</p>
    </div>
  );
};

export default Home;
