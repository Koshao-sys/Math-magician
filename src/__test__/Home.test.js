import { render } from '@testing-library/react';
import Home from '../components/Home';

it('renders correctly', () => {
  const home = render(<Home />);
  expect(home).toMatchSnapshot();
});
