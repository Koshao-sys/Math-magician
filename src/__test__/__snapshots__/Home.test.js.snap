// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
Object {
  "asFragment": [Function],
  "baseElement": <body>
    <div>
      <div
        class="home-container"
      >
        <h1>
          Welcome to our page!
        </h1>
        <p>
          The Math Magician site is a fascinating platform that allows users to performcomplex mathematical calculations with ease.The site features a user-friendlycalculator that can handle a wide range of mathematical operations, from basicaddition and subtraction to advanced trigonometry and calculus. The calculatoris highly responsive, ensuring that users can enter their equations quickly andeasily, and get instant results. But the Math Magician site is more than just acalculator. It also features a unique API that delivers inspiring and thought-provokingquotes from some of the worlds most renowned mathematicians. The quotes are displayedon the sites screen, providing users with a source of inspiration and motivation asthey work through their calculations. Whether you are a student, a teacher, or a professionalmathematician, these quotes can help you stay focused, engaged, and inspired as you tackleeven the toughest mathematical problems.One of the most impressive features of the MathMagician site is its versatility. Whether you are working on a homework assignment,preparing for an exam, or simply exploring the fascinating world of mathematics,this platform has something to offer.The calculator is highly adaptable, making itsuitable for everyone from beginners to experts, and the APIs vast collection ofquotes ensures that there is always something new and interesting to discover.Overall, the Math Magician site is a valuable resource for anyone interested inmathematics. Its calculator is highly responsive and user-friendly, while its APIprovides a wealth of inspiration and motivation for anyone looking to explore thefascinating world of mathematics. Whether you are looking to solve complex equations,brush up on your calculus, or simply immerse yourself in the beauty of mathematics,the Math Magician site is an excellent place to start.
        </p>
      </div>
    </div>
  </body>,
  "container": <div>
    <div
      class="home-container"
    >
      <h1>
        Welcome to our page!
      </h1>
      <p>
        The Math Magician site is a fascinating platform that allows users to performcomplex mathematical calculations with ease.The site features a user-friendlycalculator that can handle a wide range of mathematical operations, from basicaddition and subtraction to advanced trigonometry and calculus. The calculatoris highly responsive, ensuring that users can enter their equations quickly andeasily, and get instant results. But the Math Magician site is more than just acalculator. It also features a unique API that delivers inspiring and thought-provokingquotes from some of the worlds most renowned mathematicians. The quotes are displayedon the sites screen, providing users with a source of inspiration and motivation asthey work through their calculations. Whether you are a student, a teacher, or a professionalmathematician, these quotes can help you stay focused, engaged, and inspired as you tackleeven the toughest mathematical problems.One of the most impressive features of the MathMagician site is its versatility. Whether you are working on a homework assignment,preparing for an exam, or simply exploring the fascinating world of mathematics,this platform has something to offer.The calculator is highly adaptable, making itsuitable for everyone from beginners to experts, and the APIs vast collection ofquotes ensures that there is always something new and interesting to discover.Overall, the Math Magician site is a valuable resource for anyone interested inmathematics. Its calculator is highly responsive and user-friendly, while its APIprovides a wealth of inspiration and motivation for anyone looking to explore thefascinating world of mathematics. Whether you are looking to solve complex equations,brush up on your calculus, or simply immerse yourself in the beauty of mathematics,the Math Magician site is an excellent place to start.
      </p>
    </div>
  </div>,
  "debug": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;
